qdrant-landing/content/documentation/cloud/backups.md

```diff
@@ -1,36 +1,97 @@
 ---
 title: Backups
-weight: 30
+weight: 60
 ---
 
-# Backups
+# Cloud Backups
```

Comment:
One principle of modern technical writing is that "Every page is page one". If someone gets here from a search engine, they'll know right away that we're discussing cloud backups on this page.